tests/data/verify/poetry_group/pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.poetry]
+name = "micropipenv-test"
+version = "0.1.0"
+description = ""
+authors = ["Fridolin Pokorny <fridolin@redhat.com>"]
+
+[tool.poetry.dependencies]
+python = "^3.7"
+daiquiri = "2.0.0"
+
+[tool.poetry.group.dev.dependencies]
```

Comment:
Adding another group here would be a good test. Poetry 1.2 allows arbitrary names for groups.

So I can use the following command: `poetry add --group extra black`, which in turn would create a section "tool.poetry.group.extra.dependencies"